components/html-block/html-block.js

```diff
@@ -324,7 +334,6 @@ class HtmlBlock extends LitElement {
 
 	async _updateRenderContainer() {
 		const renderContainer = this.shadowRoot.querySelector('.d2l-html-block-rendered');
-		renderContainer.innerHTML = this.html;
 		await this._processRenderers(renderContainer);
```

Comment:
We still need to process MathJax and Prism updates post-render, so this needs to stay. I suspect this could introduce a race condition with the initial render, since these renderers will kick off as soon as `this.html` is rendered into the container (i.e. we won't wait for the `until` to resolve). So it may be necessary to make use of a promise here to ensure we don't try to process math or code snippets until our embeds have finished rendering.